### DISCUSSION TITLE
Add a way to print reminders with `test.failing`, `test.skip`, etc.

### DISCUSSION THREAD

**Discussion by tommy-mitchell** [avajs/ava#3224] [Ideas]
When using `test.failing` (and others), you might add a comment explaining why the test isn't working yet:

```js
// TODO: waiting on some/repo#123
test.failing("does foo", t => // ...
```

While this comment may show up in a lint reporter, it doesn't when running tests:

```
✔ does bar
✔ [expected fail] does foo
✔ does baz
```

Maybe the reporter could print an optional message?

```js
test.failing("does foo", t => {
	// ...
}, "waiting on some/repo#123");
```

```
✔ does bar
✔ [expected fail] does foo
  - waiting on some/repo#123
✔ does baz
```

**Comment by novemberborn** [comment]
Could you use `t.log()` within the test implementation?

**Reply by tommy-mitchell** [reply]
`t.log` does report a message with `test.failing`, but not for `test.skip` or `test.todo`.

**Reply by novemberborn** [reply]
True. We'd have to chain something:

```js
test.failing("does foo", t => t.fail()).because("waiting on some/repo#123")
```

Something like that? Or `test.because().todo()` etc.

**Reply by tommy-mitchell** [reply]
I think `test.failing().because()` reads better semantically, but it might be better in front so the reason is at the top of the test.

**Reply by novemberborn** [reply]
Thanks @tommy-mitchell, I've opened https://github.com/avajs/ava/issues/3227.
